source/extensions/filters/http/transformation/transformation_filter.cc

```diff
@@ -250,6 +251,12 @@ void TransformationFilter::transformSomething(
     void (TransformationFilter::*addData)(Buffer::Instance &)) {
 
   try {
+    // if log_request_response_info_ is set on the transformation, log the
+    // request body and request headers before transformation
+    TRANSFORMATION_SENSITIVE_LOG(debug, "headers before transformation: {}", 
+                          transformation, filter_config_, callbacks, header_map);
+    TRANSFORMATION_SENSITIVE_LOG(debug, "body before transformation: {}", 
+                          transformation, filter_config_, callbacks, body.toString());
     transformation->transform(header_map, request_headers_, body, callbacks);
```

Comment:
for convenience, do you think we will want this after the transformation too? particularly for cases where we have staged / multiple transformations I could see this being useful

Reply:
I think the other extension is to log all Extracted values as well.